pandora-client-web/src/components/wiki/pages/history.tsx

```diff
@@ -0,0 +1,112 @@
+			</p>
+
+			<h4>Does Pandora want to replace BC?</h4>
+
+			<p>
+				Not at all! Pandora has a different vision, wanting to be a secure, consensual roleplaying platform that focuses on text-heavy interactions.
+				BC was a bit like that in its first year before more and more game-like features were added that changed the focus of public rooms and the
+				interests of parts of the community slowly. Also the fact that console-usage/scripts/mods/extensions in BC are so all-powerful has certain
+				drawbacks and is exploited quite often.<br />
+				In a way, Pandora will offer an improved "classic" experience, focusing on roleplaying and clear house rules. It will not be tolerated that
+				some users can cheat or gain intransparent feature advantages that others do not have, unless they risk running external scripts.
+			</p>
+
+			<h4>What issues with BC were seen?</h4>
+
+			<p>
+				<ul>
+					<li>Technical quality of BC: The source code of BC was not of good quality and while it improved quite a bit over the years (to a large part
+						thanks to the same people who founded Pandora), it is essentially still something held together by many band aids and compromises.
+						Contributing to it is not exactly pleasant and beginner friendly. Starting from scratch seemed like a more sensible decision than
+						trying to improve it further.
+					</li>
+					<li>Not Open-Source: While the source code of BC is public, it is not licensed under any open-source license. That means that every part of BC
+						is proprietary and owned by its authors and that would mean that one cannot legally use the BC code anywhere else or copy it and further
+						develop it without the explicit permission by every person who ever contributed to BC which makes that almost impossible. The longevity of BC is
+						therefore doubtful and contributing to BC can be seen as a risk to invest time into a black hole.
+					</li>
+					<li>BC project management: The desire for a different project management approach and development process was strong.</li>
+					<li>BC server architecture: It is singular and cannot scale beyond a certain number of users, where lag and disconnects slowly get worse.
+						Starting from scratch with a modern and scalable approach seemed like the best option to give users a stable experience.
+					</li>
+					<li>
+						Lack of server validation: In many cases the server in BC does not validate what the client does, which makes it possible for everyone
+						to alter things like locks, restraints, or messages. The overall lack of security was a concern often voiced.
+					</li>
+					<li>Asset creation: Asset creation in BC needs many variants of the same image and it is an extreme effort to add new poses.</li>
+					<li>Legality of assets/images: In BC there is no process for vetoing assets. Most of them are not providing where they originate
```

Comment:
maybe: `are not providing` -> `do not provide`

Reply:
Changed, thank you.